README.md

```diff
@@ -91,74 +91,272 @@ except:
+
+```python
+from carto.sync_tables import SyncTableJobManager
+import time
+
+# how often to sync the dataset (in seconds)
+SYNC_TIME = 900
+# write here the URL for the dataset to sync
+URL_TO_DATASET = ""
+
+syncTableManager = SyncTableJobManager(auth_client)
+syncTable = syncTableManager.create(URL_TO_DATASET, SYNC_TIME)
+
+# return the id of the sync
+sync_id = syncTable.get_id()
+
+while(syncTable.state != 'success'):
+    time.sleep(5)
+    syncTable.refresh()
+    if (syncTable.state == 'failure'):
+        print('The error code is: ' + str(syncTable.error_code))
+        print('The error message is: ' + str(syncTable.error_message))
+        break
+
+# force sync
+syncTable.refresh()
+syncTable.force_sync()
 ```
 
-You can also import a dataset from a remote URL:
+**Get a list of all the current import jobs**
 
 ```python
-from carto import URLImport
+from carto.file_import import FileImportJobManager
 
-fi = URLImport(MY_URL, auth_client)
-fi.run()
+file_import_manager = FileImportJobManager(auth_client)
+file_imports = file_import_manager.all()
 ```
 
-If you specify a refresh interval (>=900s) for a remote URL, your import job becomes a sync table, and Carto will refresh the datasets based on the contents of the URL at that interval:
+**Get all the datasets**
 
 ```python
-from carto import URLImport
+from carto.datasets import DatasetManager
 
-fi = URLImport("http://test.com/myremotefile", auth_client, interval=3600)
-fi.run()
+dataset_manager = DatasetManager(auth_client)
+datasets = dataset_manager.all()
 ```
 
-At this point, ```fi.success``` indicates whether the initial upload was successful or not.
+**Get a specific dataset**
 
-You can get all the pending imports:
+```python
+from carto.datasets import DatasetManager
+
+# write here the ID of the dataset to retrieve
+DATASET_ID = ""
 
+dataset_manager = DatasetManager(auth_client)
+dataset = dataset_manager.get(DATASET_ID)
+```
+
+**Update the properties of a dataset**
 ```python
-from carto import ImportManager
+from carto.datasets import DatasetManager
+from carto.permissions import PRIVATE, PUBLIC, LINK
+
+# write here the ID of the dataset to retrieve
+DATASET_ID = ""
 
-im = ImportManager(auth_client)
-import_list = im.all(ids_only=False)
+dataset_manager = DatasetManager(auth_client)
+dataset = dataset_manager.get(DATASET_ID)
+
+# make the dataset PUBLIC
+dataset.privacy = PUBLIC
+dataset.save()
 ```
 
-Or just one:
+**Delete a dataset**
 
 ```python
-im = ImportManager(auth_client)
-single_import = im.get("afaab071-dc95-4bda-a772-ea37f8729157")
+from carto.datasets import DatasetManager
+
+# write here the ID of the dataset to retrieve
+DATASET_ID = ""
+
+dataset_manager = DatasetManager(auth_client)
+dataset = dataset_manager.get(DATASET_ID)
+dataset.delete()
 ```
 
-You can update the attributes of an import job any time (like for checking if an import has finished):
+**Export a CARTO visualization**
```

Comment:
While we decide what to do, I've added warnings when using any non-public API and added warnings in the API docs as well.

From now on I continue working on branch `1.0.0`